_posts/2018-11-21-large-touch-targets.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Quick test: Large touch targets"
+description: "How to determine if your touch targets are large enough to be used."
+author: eric_bailey
+published: true
+categories:
+  - Quick Tests
+---
+
+A touch target is the total area a user can click or tap on to activate an interactive control such as a link, input, or button.
+
+## Background
+
+Ensuring that interactive elements have sufficiently large touch targets helps people with [motor control issues](https://webaim.org/articles/motor/motordisabilities) use your website or web app. Like trying to thread a needle, providing a too-small touch target may make it difficult or impossible for some people to be able to reliably reach the area required activate the control.
```

Comment:
Suggest:
"Ensuring that interactive elements have sufficiently sized touch targets can help people..."

Instead of saying "website or web app" maybe just say "website or application".  generalize it a bit as it can be useful for native apps too.

Suggest:
"Like trying to thread a needle, providing a touch targets that are too-small may make..."
"...reliably activate the interactive element."

(that last bit of the sentence seemed like you might have been writing something else?  Also keeping consistent with interactive element vs control in the same paragraph)

Reply:
I rephrased the threading the needle bit, does it read better to you now?

Also consolidated the element/control usage, thanks for that.